pkg/config/sdk_test.go

```diff
@@ -34,22 +33,22 @@ func TestSDKInfo_UnmarshalJSON(t *testing.T) {
 	}
 }
 
-func TestSDKInfo_AnnotationInvalid(t *testing.T) {
-	b, err := os.ReadFile("../../test/res/config.json")
-	if err != nil {
-		t.Fatalf(err.Error())
-	}
-
-	var cfg SdkInfo
-	err = json.Unmarshal(b, &cfg)
-	if err != nil {
-		t.Fatalf(err.Error())
-	}
-
-	cfg.Annotators = append(cfg.Annotators, "invalid")
-	b, _ = json.Marshal(cfg)
-
-	var x SdkInfo
-	err = json.Unmarshal(b, &x)
-	test.CheckError(err, true, "test sdk invalid annotation", t)
-}
+// func TestSDKInfo_AnnotationInvalid(t *testing.T) {
```

Comment:
@michaelehab We should just remove this function. Its original purpose was to check the known list of annotator keys prior to introducing "custom annotators"